tests/conftest.py

```diff
@@ -43,6 +43,12 @@ def mock_slurmrestd_api_cache_dir(tmp_path):
         yield _cache_dir
 
 
+@pytest.fixture(autouse=True)
+def slurmrestd_jwt_key_path(tmp_path):
+    _jwt_dir = tmp_path / "jwt.key"
```

Comment:
Do you think you should call `mkdir()` on this path here?

Reply:
hmm good point. I create the dir where the fixture is used instead in the fixture itself. I guess it makes sense to change. I shall do it

Reply:
oh, I just noticed, I decided to create the dir in the test itself because the secret key string is defined in the test and I needed that to write in the file

Reply:
resolving since I changed